packer/packer.json

```diff
@@ -90,7 +90,8 @@
       "type": "ansible",
       "playbook_file": "../ansible/playbook.yml",
       "ansible_env_vars": [
-        "ANSIBLE_SSH_ARGS='{{user `existing_ansible_ssh_args`}} -o IdentitiesOnly=yes'"
+        "ANSIBLE_SSH_ARGS='{{user `existing_ansible_ssh_args`}} -o IdentitiesOnly=yes'",
+        "ANSIBLE_REMOTE_TEMP='/tmp/.ansible/'"
```

Comment:
This could potentially cause issues with custom base AMIs if they configure `/tmp` with `noexec`, I'm not sure if that is something we need to be overly concerned about, though.

Reply:
We can always change it later if necessary, or create the temporary folder under the user's directory, although it seems that ansible isn't cleaning that up afterwards.

Reply:
Sounds good, can you create an issue to track that?